doc/nrf/ug_zigbee_cli_reference.rst

```diff
@@ -0,0 +1,620 @@
+
+   bdb channel <n>
+
+.. note::
+    |precondition2|
+
+If the optional argument is not provided, get the current number and bitmask of the channel.
+
+If the optional argument is provided:
+
+* If ``n`` is in ``[11:26]`` range, set to that channel.
+* Otherwise, treat ``n`` as bitmask (logical or of a single bit shifted by channel number).
+
+
+Example:
+
+.. code-block::
+
+   > bdb channel 0x110000
+   Setting channel bitmask to 110000
+   Done
+
+----
+
+.. _bdb_ic:
+
+bdb ic
+******
+
+Set install code on the device, add information about the install code on the trust center, set the trust center install code policy.
+
+.. code-block::
+
+   bdb ic add <h:install code> <h:eui64>
+   bdb ic set <h:install code>
+   bdb ic policy <enable|disable>
+
+.. note::
+    |precondition3|
+
+* ``bdb ic set`` must only be used on a joining device.
+
+* ``bdb ic add`` must only be used on a coordinator.
+  For ``<h:eui64>``, use the address of the joining device.
+
+* ``bdb ic policy`` must only be used on a coordinator.
+
+Provide the install code as an ASCII-encoded :file:`HEX` that includes CRC16.
+
+For production devices, an install code must be installed by the production
+configuration present in flash.
+
+
+Example:
+
+.. code-block::
+
+   > bdb ic add 83FED3407A939723A5C639B26916D505C3B5 0B010E2F79E9DBFA
+   Done
+
+
+----
+
+.. _bdb_legacy:
+
+bdb legacy
+**********
+
+Enable or disable the legacy device support.
+
+.. code-block::
+
+   bdb legacy <enable|disable>
+
+Allow or disallow legacy pre-r21 devices on the Zigbee network.
+
+Example:
+
+.. code-block::
+
+   > bdb legacy enable
+   Done
+
+----
+
+.. _bdb_nwkkey:
+
+bdb nwkkey
+**********
+
+Set network key.
+
+.. code-block::
+
+   bdb nwkkey <h:key>>
+
+Set a pre-defined network key instead of a random one.
+
+.. note::
+    |precondition2|
+
+Example:
+
+.. code-block::
+
+   > bdb nwkkey 00112233445566778899aabbccddeeff
+   Done
+
+----
+
+.. _bdb_factory_reset:
+
+bdb factory_reset
+*****************
+
+Perform a factory reset via local action.
+See Base Device Behavior specification chapter 9.5 for details.
+
+.. code-block::
+
+   > bdb factory_reset
+   Done
+
+----
+
+.. _bdb_child_max:
+
+bdb child_max
+*************
+
+Set the amount of child devices that is equal to <d:nbr>.
+
+.. code-block::
+
+   > bdb child_max <d:nbr>
+
+.. note::
+    |precondition2|
+
+Example:
+
+.. code-block::
+
+   > bdb child_max 16
+   Setting max children to: 16
+   Done
+
+----
+
+.. _zdo_simple_desc_req:
+
+zdo simple_desc_req
+*******************
+
+Send Simple Descriptor Request.
+
+.. code-block::
+
+   zdo simple_desc_req <h:16-bit destination address> <d:endpoint>
+
+Send Simple Descriptor Request to the given node and endpoint.
+
+Example:
+
+.. code-block::
+
+   > zdo simple_desc_req 0xefba 10
+   > src_addr=0xEFBA ep=0x260 profile_id=0x0102 app_dev_id=0x0 app_dev_ver=0x5
+     in_clusters=0x0000,0x0003,0x0004,0x0005,0x0006,0x0008,0x0300 out_clusters=0x0300
+   Done
```

Comment:
Just a general note, using Shell to print command response does not prepend the response with any prompt
```suggestion
   > zdo simple_desc_req 0xefba 10
   src_addr=0xEFBA ep=0x260 profile_id=0x0102 app_dev_id=0x0 app_dev_ver=0x5
   in_clusters=0x0000,0x0003,0x0004,0x0005,0x0006,0x0008,0x0300 out_clusters=0x0300
   Done
```